docs/connecting-your-data/assignment-tables/randomization-sdk.md

```diff
@@ -0,0 +1,163 @@
+
+```python
+def get_background_color(user):
+	variation = get_variant(user)
+	if variation == 'treatment':
+		return 'DARK'
+	else:
+		return 'WHITE'
+```
+
+Eppo’s SDK handles the variation assignment for you. Given a user subject and experiment, the SDK assigns a variation based on your experiment traffic allocation and variations configured in Eppo. If the traffic allocation changes, subjects already assigned to a variation are not reassigned.
+
+# Getting Started
+
+## Generating an API Token
+
+You will need to generate an API token that allows your application to communicate with Eppo’s experiments API. Do not share your API token with anyone outside your organization.
+
+### 1. Navigate to the **Admin** tab in the Eppo homepage and click **Create Key**
+![generating-api-token-1](../../../static/img/connecting-data/api-key-1.png)
+
+### 2. Fill in a unique name for the key and give it read access
+![generating-api-token-2](../../../static/img/connecting-data/api-key-2.png)
+
+### 3. Click **Create**
+
+A modal will pop up with the API key -- download it or copy it somewhere else for safe-keeping.
+![generating-api-token-3](../../../static/img/connecting-data/api-key-3.png)
+
+### 4. Close your modal to see your API token
+![generating-api-token-4](../../../static/img/connecting-data/api-key-4.png)
+
+## Install the SDK package in your preferred language
+
+:::note
+Eppo’s Node SDK is meant for use in server applications only. It is not compatible with browser or client JS applications.
+:::
+
+<Tabs>
+<TabItem value="node" label="Node">
+
+```bash
+yarn add @eppo/node-server-sdk
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```bash
+pip install eppo-server-sdk
+```
+</TabItem>
+</Tabs>
+
+
+## Initializing the Eppo Client in your application code
+
+Initialize the SDK once when your application starts up to generate a shared client instance. Upon initialization, the SDK will begin polling Eppo’s API at regular intervals to retrieve your experiment configurations.
+
+:::note
+The client must be a singleton. The client instance stores assignment configurations in memory. The same client instance should be reused for the lifetime of your application. Do not generate a new instance on every request.
+:::
+
+<Tabs>
+<TabItem value="node" label="Node">
+
+```tsx
+import * as EppoSdk from '@eppo/node-server-sdk';
+
+const eppoClient = EppoSdk.init({ apiKey: 'YOUR_API_KEY' });
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+import eppo_client
+from eppo_client.config import Config
+
+client_config = Config(api_key="<YOUR_API_KEY>")
+client = eppo_client.init(client_config)
+```
+
+</TabItem>
+</Tabs>
+
+
+## Retrieving Assignment Configurations
+
+Each SDK client exposes a variation assignment function. This function takes 2 inputs:
+
+- `experimentKey` - this is the value from the “Feature Flag” input on the Experiment “Setup” tab as seen in the below screenshots
```

Comment:
The screenshot here needs to be updated since the UI was renamed from "Feature Flag" -> "Experiment Key"